<launch>

  <include file="$(find m5stack_ros)/launch/timer_cam.launch">
	<arg name="port" value="/dev/rfcomm0" />
	<arg name="namespace" value="timercam_1" />
    <arg name="camera_info" value="timercam_1_info.yaml" />
  </include>
  <include file="$(find m5stack_ros)/launch/timer_cam.launch">
	<arg name="port" value="/dev/rfcomm1" />
	<arg name="namespace" value="timercam_2" />
    <arg name="camera_info" value="timercam_2_info.yaml" />
  </include>
</launch>
